health-checker/pom.xml

```diff
@@ -198,6 +198,9 @@
         <configuration>
           <archive>
             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            <manifestEntries>
+              <x-Stargate-Health-Checkers>deadlocks</x-Stargate-Health-Checkers>
```

Comment:
"deadlocks"?

Reply:
comes from the framework, I guess (did not chase it down). I thought it would be useful to enforce by default.

Reply:
it checks for java thread deadlocks.

Reply:
Oh, interesting.